shoop/core/order_creator/_source.py

```diff
@@ -404,6 +420,13 @@ def get_validation_errors(self):
             for error in payment_method.get_validation_errors(source=self):
                 yield error
 
+        for supplier in self.get_suppliers():
+            for product, quantity in iteritems(self.get_products_and_quantities(supplier)):
+                shop_product = product.get_shop_instance(shop=self.shop)
+                for error in shop_product.get_orderability_errors(
+                        supplier=supplier, quantity=quantity, customer=self.customer):
+                    yield error
+
```

Comment:
I think that checking that there is shipping and payment methods is just for BaseBasket. So I leave those there at least for now. ShopProduct checks after those is now unneccesary since it's already done in OrderSource validation, so I will remove shop product orderability checks from BaseBasket.